usrsctplib/user_socket.c

```diff
@@ -3285,6 +3285,10 @@ usrsctp_conninput(void *addr, const void *buffer, size_t length, uint8_t ecn_bit
 	return;
 }
 
+void usrsctp_fire_timer(int delta)
```

Comment:
Nit: I would prefer usrsctp_handle_timers() as the function name.
Can we also make the argument an unsigned int?